tests/excerptexport/views/test_export_list.py

```diff
@@ -0,0 +1,61 @@
+from unittest.mock import patch
+
+from django.contrib.auth.models import User
+from django.contrib.gis import geos
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+
+from osmaxx.excerptexport.models import Excerpt, ExtractionOrder
+from tests.excerptexport.permission_test_helper import PermissionHelperMixin
+
+
+# FIXME: test more possibilities
```

Comment:
That'd be a `TODO`, not a `FIXME`. It's not (necessarily) broken without more tests, is it?